src/v2/Components/NavBar/NavItem.tsx

```diff
@@ -1,10 +1,70 @@
-import { Box, BoxProps, Link, Sans, space } from "@artsy/palette"
+import {
+  Box,
+  BoxProps,
+  Clickable,
+  Flex,
+  Link,
+  Sans,
+  color,
+} from "@artsy/palette"
 import { AnalyticsSchema } from "v2/Artsy"
 import { useTracking } from "v2/Artsy/Analytics/useTracking"
 import { isFunction, isString } from "lodash"
-import React, { useState } from "react"
+import React, { useEffect, useRef, useState } from "react"
 import { animated, config, useSpring } from "react-spring"
-import styled from "styled-components"
+import styled, { css } from "styled-components"
+import { position } from "styled-system"
+
+const Container = styled(Flex)`
+  height: 100%;
+  align-items: center;
+`
+
+const HitArea = styled(Link)`
```

Comment:
Same here

Reply:
[Rather than duplicate these styles](https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Link/Link.tsx#L43-L50) I just am handling this like `as={Menu ? Clickable : RouterLink}`